addons/hr_work_entry_contract_attendance/__init__.py

```diff
@@ -0,0 +1,85 @@
+#-*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+from odoo import api, SUPERUSER_ID
+from . import models
+from . import wizard
+
+def _generate_attendances(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+
+    hne_contract = env.ref('hr_work_entry_contract_attendance.hr_contract_hne', raise_if_not_found=False)
+    if hne_contract:
+        if not env['hr.attendance'].sudo().search_count([('employee_id', '=', hne_contract.employee_id.id)]):
```

Comment:
```py
if not hne_contract or not not env['hr.attendance'].sudo().search_count([('employee_id', '=', hne_contract.employee_id.id)]):
    return
```
to avoid 2 tabs ?